packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -52,6 +53,18 @@ describe('Sayt Component', () => {
       expect(setSearchboxListener).to.be.calledWith(searchbox, 'add');
       expect(addEventListener).to.be.calledWith(HOVER_AUTOCOMPLETE_TERM_EVENT, sayt.handleAutocompleteTermHover);
     });
+
+    it('should call `debounceSaytRequests` for each of the two debounced methods', () => {
```

Comment:
We haven't been using backticks in our test descriptions as far as I know (I've also added () below):
```suggestion
    it('should call debounceSaytRequests() for each of the two debounced methods', () => {
```